### ISSUE TITLE
Error creating remote relationship on 2.1.0

### ISSUE THREAD

**Issue by qstearns** [hasura/graphql-engine#7933]
### Version Information

Server Version: 2.1.0
CLI Version (for CLI related issue): 2.1.0

### Environment

OSS (Docker on M1 Mac)

### What is the expected behaviour?

I'm hoping to create a remote relationship from the console
### Keywords

error creating remote relationship
key definition not found

### What is the current behaviour?

Currently Hasura returns an api error:
```
{
    "path": "$.args[0].args",
    "error": "Error when parsing command create_remote_relationship.\nSee our documentation at https://hasura.io/docs/latest/graphql/core/api-reference/metadata-api/index.html#metadata-apis.\nInternal error message: key \"definition\" not found",
    "code": "parse-failed"
}
```

### How to reproduce the issue?

1. In Hasura 2.1.0, create a remote relationship from the console.

### Please provide any traces or logs that could help here.

The error message returned by the server:

```
{
    "path": "$.args[0].args",
    "error": "Error when parsing command create_remote_relationship.\nSee our documentation at https://hasura.io/docs/latest/graphql/core/api-reference/metadata-api/index.html#metadata-apis.\nInternal error message: key \"definition\" not found",
    "code": "parse-failed"
}
```

### Any possible solutions?

It seems like the console may be hitting the server with an invalid API call.

### Can you identify the location in the source code where the problem exists?

I cannot, but it seems to be related to #7906 and #7916

### If the bug is confirmed, would you be willing to submit a PR?

I cannot Haskell, and realistically it seems like someone might beat me to it, but I guess I'd give it a shot.

**Comment by qstearns** [comment]
The issue in the console can also be circumvented by defining the relationship in metadata and applying there.

**Comment by tirumaraiselvan** [comment]
Apart from the main issue,  confusion stems from the fact that the error message reported by the API is slightly incorrect. See attached screenshot below:

![Screen Shot 2021-12-15 at 9 36 07 AM](https://user-images.githubusercontent.com/8663570/146121361-d5298d60-54cc-482d-bfcf-a904a3dd3348.png)

The console is making the `pg_create_remote_relationship` API but the server gives the error message with `create_remote_relationship` API.

**Comment by tirumaraiselvan** [comment]
Released in v2.1.1
